README.md

```diff
@@ -116,8 +116,10 @@ then Lodash/Underscore is the better option.*
 1. [_.range](#_range)
 1. [_.reduce](#_reduce)
 1. [_.reduceRight](#_reduceright)
+1. [_.orderBy](#_sortby-and-_orderby)
```

Comment:
A tiny last change - need move this _.orderBy to up, after _.minBy and before _.plunk ... ( as it's alphabetically listed)

Reply:
Ah shoot, moved it to the right spot.